src/ATen/native/xpu/XPUFallback.template

```diff
@@ -184,7 +184,6 @@ TORCH_LIBRARY_IMPL(aten, XPU, m) {
     "_linalg_svd.U",
     "lu_unpack.out",
     "ormqr",
-    "_scaled_dot_product_efficient_attention",
```

Comment:
Removed as there is on CPU implementation of `_scaled_dot_product_efficient_attention`